src/functionaltests/java/com/ericsson/ei/notifications/trigger/SubscriptionNotificationSteps.java

```diff
@@ -187,6 +190,14 @@ public void send_one_previous_event() throws Throwable {
         eventManager.sendEiffelEvents(EIFFEL_EVENTS_JSON_PATH, eventNamesToSend);
     }
 
+    @Then("^Default values mail notification are assigned$")
+    public void default_values_mail_notification() {
+        if (sender == null || subject == null) {
+            sender = "noreply@ericsson.com";
+            subject = "Email subscription notification";
+        }
+    }
```

Comment:
Please update step "Mail subscriptions were triggered" with the method name "mail_subscriptions_were_triggered" to also validate the subject in the email, currentluy it only makes an assert on the "From" field that is the sender.

You should also test and ensure that it is possible to give only sender and not subject and the given sender should still be the value given by the user, and the other way around, a subject is given by the user but not sender.